Makefile

```diff
@@ -1,11 +1,17 @@
 CXX = g++ -D_GLIBCXX_USE_CXX11_ABI=0 -std=c++11
 CXXFLAGS = -g -O3
-LDFLAGS = -g -static
+LDFLAGS = -g
```

Comment:
This is great, I hate it when I need to compile on OS X and comment the line out.